### PR TITLE
fix: Missing rest of sales/lists on collection while using filter

### LINKED ISSUE #9048: Missing rest of sales/lists on collection while using filter
https://canary.kodadot.xyz/ahp/collection/66/activity?sale=true

you can see there is definitely more than 3 sales by chart

image

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

  👇 __ Let's make a quick check before the contribution.

  ## PR Type

  - [x] Bugfix

  ## Needs QA check

  - @kodadot/qa-guild please review

  ## Context

  - [x] Closes #9048

  #### Did your issue had any of the "$" label on it?

  - [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

  ## Screenshot 📸

  - [x] My fix has changed UI

image

  ## Copilot Summary
  copilot:summary

  copilot:poem

### CODE REVIEW

File: components/collection/activity/events/Events.vue [hunk 1]
```diff
@@ -96,15 +96,17 @@ const filteredEvents = computed(() => {
...
-      .map(getIdentityId)
-      .some((x) => identityIds.includes(x))
-
-    if (!isAnyEventTypeFilterActive() && filterByVerifiedIdentity) {
-      return active
+    const isActiveEvent = filterByVerifiedIdentity
+      ? getEventAddresses(event)
+          .map(getIdentityId)
+          .some((x) => identityIds.includes(x))
+      : true
```
Comment (roiLeo) [inline components/collection/activity/events/Events.vue:103]: hmmm.. always return true if false
any way to refactor it for better readability?

**Comment by yangwao** [comment]
thanks!
pay 20 usd

File: components/collection/activity/events/Events.vue (hunk 1 above)
Comment (yangwao) [reply components/collection/activity/events/Events.vue:103]: back log please

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 6.66 USD/DOT ~ 3.003 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0x06d0a040a02514f8581a614ca09e4f24aaffe141c78cbddf1f6775b21702bab9](https://polkadot.subscan.io/extrinsic/0x06d0a040a02514f8581a614ca09e4f24aaffe141c78cbddf1f6775b21702bab9)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**